.byebug_history

```diff
@@ -0,0 +1,95 @@
+c
```

Comment:
This file shouldn't be comitted

Reply:
resolved https://github.com/discourse/discourse/pull/27155/commits/7c837ba1711c812bff6b635038b2e5ea18d06406